README_fr-FR.md

```diff
@@ -0,0 +1,443 @@
+# Guide sur les messages de commit
+
+[![Dites merci!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/RomuloOliveira)
+
+Un guide pour comprendre l'importance des messages de commit et comment bien les écrire.
+
+Cela pourrait vous aider à comprendre ce qu'un commit est, pourquoi c'est important de bien écrire ses messages, les meilleurs techniques et quelques astuces, pour mieux (ré)écrire une bonne "commit history" 
+
+## Langages Disponibles
```

Comment:
Since #31 there is no need to include this section anymore. Could you add in the [README.md](https://github.com/RomuloOliveira/commit-messages-guide/blob/master/README.md) intead?